vector/src/main/java/im/vector/services/EventStreamService.java

```diff
@@ -1352,16 +1353,7 @@ public void run() {
 
                             // the notification cannot be built
                             if (null != notif) {
-                                if (shouldDisplayListenForEventsNotification()) {
-                                    mIsForeground = true;
-                                    startForeground(NOTIFICATION_ID, notif);
-                                } else {
-                                    if (mIsForeground) {
-                                        stopForeground(true);
-                                        mIsForeground = false;
-                                    }
-                                    nm.notify(NOTIFICATION_ID, notif);
-                                }
+                                nm.notify(NOTIFICATION_ID, notif);
```

Comment:
the plan was to merge the both notifications in a single one
1- many fdroid users complained about it
2- on google play version, during the GCM registration, the service must be put in foreground to trigger notifications.

As yous said in the android room, the froid users are used to have both notifications (at least some of them).
But, we have to have a "simpler" behaviour for the google play users. Many of them already complained about this "listen for events" notification.

Reply:
FWIW, I think the previous behavior is less 'simple' as it leads to people thinking the app is broken (since it produces un-swipable notifications). 

As for point 2, shouldn't the 'Listen for events' notification come up if GCM registration fails regardless of this section? Ideally, people on google play should never see the 'listening for events' notification, so I don't think it's worth it to make a special case for if a user is on google play but dosen't have GCM. 

If you think otherwise though, I can restore the previous behavior on google play when GCM fails, what do you think?